expression/builtin_compare.go

```diff
@@ -855,17 +864,18 @@ func (c *intervalFunctionClass) getFunction(ctx sessionctx.Context, args []Expre
 	}
 	var sig builtinFunc
 	if allInt {
-		sig = &builtinIntervalIntSig{bf}
+		sig = &builtinIntervalIntSig{bf, hasNullable}
 		sig.setPbCode(tipb.ScalarFuncSig_IntervalInt)
```

Comment:
We need to add new pbcode for the sigs when hasNullable is true.
And we can use the new pbcode [here](https://github.com/pingcap/tidb/pull/19543/files#diff-8fae0b14f3eaaced7305742a19673d50R227) to check.